web/src/app/projects/view/view.component.ts

```diff
@@ -123,6 +151,17 @@ export class ViewProjectComponent implements OnInit, OnDestroy {
     return this.project.isAdmin(this.authService.profile.uid);
   }
 
+  /**
+   * Set the followers flag true if user uid is present in the database else set false
+   */
+  public setFollower(): void {
+    if (this.authService.profile.following.includes(this.project.uid)) {
```

Comment:
User might be a guest and not logged in, `this.authService.profile.following` can be `undefined`

Reply:
Fixed by putting a default on the `ProfileModel` 

```typescript
following?: string[] = [];
```